linked_list/linked-list.js

```diff
@@ -10,11 +10,64 @@ class Node{
 class LinkedList{
   constructor(){
     this.head = null;
+    this.itemCount = 0;
   }
   insert(val){
     let newHead = new Node(val);
     newHead.next = this.head;
     this.head = newHead;
+    this.itemCount++;
+  }
+
+  append(val){
```

Comment:
While adding an itemCount is not an issue. If you want more practice with DSA try writing the methods without itemCount